data/json/items/tool_armor.json

```
@@ -2528,6 +2528,90 @@
     "covers": [ "head", "mouth", "torso", "arm_l", "arm_r", "leg_l", "leg_r", "hand_l", "hand_r", "foot_l", "foot_r" ],
     "warmth": 60
   },
+  {
+    "id": "turtleneck",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "turtleneck" },
+    "description": "A tasteful wool turtleneck, modeled after cashmere.  Unemcumbering and warm, but lacking storage.",
+    "weight": "478 g",
+    "volume": "2500 ml",
+    "price": 5000,
+    "material": [ "wool" ],
+    "symbol": "[",
+    "looks_like": "sweater",
+    "color": "white",
+    "covers": [ "torso", "arm_l", "arm_r", "mouth" ],
+    "coverage": 95,
+    "encumbrance": 8,
+    "warmth": 35,
+    "material_thickness": 2,
+    "use_action": { "target": "turtleneck_rolled", "msg": "You adjust the neck and collar of the turtleneck.", "type": "transform" },
+    "flags": [ "VARSIZE", "FANCY", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "turtleneck_rolled",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "turtleneck (rolled)" },
+    "description": "This wool turtleneck's neck is rolled out of the way of the mouth.",
+    "weight": "478 g",
+    "volume": "2500 ml",
+    "price": 5000,
+    "material": [ "wool" ],
+    "symbol": "[",
+    "looks_like": "sweater",
+    "color": "white",
+    "covers": [ "torso", "arm_l", "arm_r" ],
+    "coverage": 95,
+    "encumbrance": 8,
+    "warmth": 35,
+    "material_thickness": 2,
+    "use_action": { "target": "turtleneck", "msg": "You adjust the neck and collar of the turtleneck.", "type": "transform" },
+    "flags": [ "VARSIZE", "FANCY", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "turtleneck_shirt",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "turtleneck" },
```

```suggestion
    "name": { "str": "turtleneck shirt" },
```
Otherwise it has the same name as "turtleneck" and it's too easy to mix up those items.